db/migrate/20141010162405_create_front_end_builds_builds.rb

```diff
@@ -1,4 +1,4 @@
-class CreateFrontEndBuildsBuilds < ActiveRecord::Migration
+class CreateFrontEndBuildsBuilds < ActiveRecord::Migration[4.2]
```

Comment:
I don't think this matters too much since you're editing migrations, but why is it using 4.2 instead of 6.* or even 5.*?

Reply:
In Rails 5 some changes were made to how migrations are run that makes them dependent on the Rails version. Migrations from Rails 4.2 or below all run the same way. We don't need to think about this most of the time, because when Rails 5 was introduced, the version is automatically added for us.

All of these migrations are from 6+ years ago so they are probably pre-Rails 4 even. But, since all migrations before 4.2 are compatible with 4.2, I chose to use that as the version. 

These are all pretty basic migrations, stuff like adding columns, so honestly it would probably work fine with any of the Rails versions. But I don't think it's worth spending the time to check that they're compatible with modern Rails versions, particularly because this gem is deprecated.